stacks_queues/stack.py

```diff
@@ -9,30 +9,21 @@ def __init__(self):
         self.store = LinkedList()
 
     def push(self, element):
-        """ Adds an element to the top of the Stack.
-            Returns None
-        """
-        pass
+       self.store.add_last(element)
 
     def pop(self):
-        """ Removes an element from the top
-            Of the Stack
-            Raises a StackEmptyException if
-            The Stack is empty.
-            returns None
-        """
-        pass
+        if self.store.length == 0:
+            raise StackEmptyException
+        else:
+            return self.store.remove_last()
 
     def empty(self):
```

Comment:
✨ Nice, alternatively note that that `LinkedList` also has an `empty` method you could take advantage of here